ThreadPoolManagerTest unstable

This test failed in a GHA Windows build: https://github.com/wborn/openhab-core/actions/runs/3758246710/jobs/6386349876

```
[ERROR] Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 3.194 s <<< FAILURE! - in org.openhab.core.common.ThreadPoolManagerTest
[ERROR] org.openhab.core.common.ThreadPoolManagerTest.testGetPoolShutdown  Time elapsed: 1.57 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Checking if thread pool Test works ==> expected: <true> but was: <false>
	at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)
	at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:210)
	at org.openhab.core.common.ThreadPoolManagerTest.checkThreadPoolWorks(ThreadPoolManagerTest.java:128)
	at org.openhab.core.common.ThreadPoolManagerTest.testGetPoolShutdown(ThreadPoolManagerTest.java:108)
```

--- a/bundles/org.openhab.core/src/test/java/org/openhab/core/common/ThreadPoolManagerTest.java
+++ b/bundles/org.openhab.core/src/test/java/org/openhab/core/common/ThreadPoolManagerTest.java
@@ -125,7 +125,7 @@ public class ThreadPoolManagerTest {
         ExecutorService threadPool = ThreadPoolManager.getPool(poolName);
         CountDownLatch cdl = new CountDownLatch(1);
         threadPool.execute(cdl::countDown);
-        assertTrue(cdl.await(1, TimeUnit.SECONDS), "Checking if thread pool " + poolName + " works");
+        assertTrue(cdl.await(5, TimeUnit.SECONDS), "Checking if thread pool " + poolName + " works");
         assertFalse(threadPool.isShutdown(), "Checking if thread pool is not shut down");
     }
 
@@ -133,7 +133,7 @@ public class ThreadPoolManagerTest {
         ScheduledExecutorService threadPool = ThreadPoolManager.getScheduledPool(poolName);
         CountDownLatch cdl = new CountDownLatch(1);
         threadPool.schedule(cdl::countDown, 100, TimeUnit.MILLISECONDS);
-        assertTrue(cdl.await(1, TimeUnit.SECONDS), "Checking if thread pool " + poolName + " works");
+        assertTrue(cdl.await(5, TimeUnit.SECONDS), "Checking if thread pool " + poolName + " works");
         assertFalse(threadPool.isShutdown(), "Checking if thread pool is not shut down");
     }
 }

